Dockerfile

```diff
@@ -218,8 +223,11 @@ COPY bin bin
 COPY data data
 
 USER root
-RUN chown -R ubuntu:ubuntu /usr/src/app
-USER ubuntu
+RUN chown -R ubuntu:0 ${APP_ROOT} ${HOME} && \
+  chmod -R g=u ${APP_ROOT} ${HOME} /etc/passwd
+
+# Numeric user ID for the ubuntu user. Used to indicate a non-root user to OpenShift
+USER 1000
```

Comment:
Same as `USER ubuntu` except using the numeric user ID for the ubuntu user.

>Lastly, the final USER declaration in the Dockerfile should specify the user ID (numeric value) and not the user name. This allows OpenShift Container Platform to validate the authority the image is attempting to run with and prevent running images that are trying to run as root, because running containers as a privileged user exposes potential security holes.

https://docs.openshift.com/container-platform/3.6/creating_images/guidelines.html

Reply:
Until now I was concerned that this openshift adaptation was making Renovate's image less secure, but it seems their approach is intended to make it more secure?

Reply:
Absolutely. OpenShift attempts to make it very difficult to run code with extended privileges.